os/include/tinyara/bluetooth/iob/iob.h

```diff
@@ -66,6 +66,7 @@
 #endif
 
 /* Some I/O buffers should be allocated */
+#define CONFIG_IOB_NBUFFERS 20
```

Comment:
In TizenRT CONFIG_* should be generated in Kconfig. you can't use 'CONFIG_' prefix to macro which defined in source codes. So move it to Kconfig if it need to be changed

Reply:
Moved the  CONFIG_IOB_NBUFFERS to Kconfig